windsurf-desktop/README.md

```diff
@@ -0,0 +1,35 @@
+---
+display_name: Windsurf IDE
+description: Add a one-click button to launch Windsurf IDE
+icon: ../.icons/windsurf.svg
+maintainer_github: coder
+verified: true
```

Comment:
I'm not, just enjoy the IDE and figured it would be easy to make one of these integrations since it's a vscode based project. But I guess the licenses around using the official extensions marketplace might make things more difficult